include/deal.II/lac/precondition.h

```diff
@@ -3623,7 +3633,8 @@ PreconditionChebyshev<MatrixType, VectorType, PreconditionerType>::
 
           // include a safety factor since the CG method will in general not
           // be converged
-          info.max_eigenvalue_estimate = 1.2 * eigenvalue_tracker.values.back();
+          info.max_eigenvalue_estimate = (data.use_fourth_kind ? 1.1 : 1.2) *
+                                         eigenvalue_tracker.values.back();
```

Comment:
Maybe we should remove this change for now; I did a few tests only where it worked well, better than for the first kind, but more tests are needed to say for sure.